spec/models/organization_spec.rb

```diff
@@ -8,4 +8,43 @@
 
   it_should_behave_like 'notable'
 
+  it 'should set and get custom values' do
```

Comment:
I considered creating a shared example group, but it seemed that there were too many context parameters which would need to be passed in and the abstractness would result in test code more difficult to follow.  In this case a set of tests specific to Organization felt more useful.  

Creating an additional shared example group could still be done, although I'm trying to balance the effort invested into more rigorous unit tests versus the pressure to keep within the project budget.